modules/special-rules/pages/shooting-with-war-engines.adoc

```diff
@@ -1,2 +1,58 @@
 = Shooting with war engines
 
+When you reach the "Shoot with war engines" step in the turn sequence then it is time to shoot with your War Engine units.
+This is usually a moment of high drama and excitement -- will your carefully positioned behemoths live up to their fearsome reputations and lay waste to the enemy?
+
+== Weapon mounts
+
+Every War Engine unit has a number of weapon mounts.
+We list each of these in a separate line on the unit's data sheet.
+Usually, each weapon mount holds one weapons system, either a super-heavy weapon or some set of weapons that has a basic Firepower value.
+
+* While you may split fire to target different detachments with different weapon mounts, you may not split fire with the effects of a single weapon mount.
+* For critical damage results, apply effects that destroy or restrict a weapon to the whole weapon mount.
+{blank}[TODO: If we reference 'weapon mount' rather than 'weapon' on every data sheet in future, perhaps we can do away with this bullet point?]
+* Where a weapon mount holds weapons that do not have a basic Firepower value, treat it as a single super-heavy weapon and apply all the rules as normal for a single super-heavy weapon:
++
+include::partial$summary-of-super-heavy-weapon-effects.adoc[]
+
+[NOTE.e40k]
+====
+_Epic 40,000_ mentions 'weapon mounts' but does not define the term or use it consistently.
+We've integrated various references to make a robust definition that helps us to make certain rules more clear and precise.
+====
+
+== Weapon arcs
+
+Each weapon mount on a War Engine unit has an arc that constrains the line of fire for weapons in that mount.
+You may only use weapons that are on that mount against targets that are within that arc.
+We use icons on the War Engine data sheets to show what arc applies to each weapon mount.
+
+.Anatomy of a weapon arc icon
+image::fire-arcs-explanation.png["Weapon arcs explanation",320]
+The arrow in the centre of each icon shows the facing of the war engine.
+The black segment shows the weapon arc of the mount.
+
+.Weapon arc icons
+// [TODO: We might want to agree on different terms and change those I've inserted below.]
+image::fire-arcs-expanded.png["Weapon arc icons",352]
+
+From top-left to bottom-right, these icons represent:
+
+[none]
+* diagonal left (180º); ahead (90º); front (180º); diagonal right (180º)
+* left (180º); all round (360º); rear (180º); right (180º)
+
+.Weapon arcs
+
+[NOTE.e40k]
+====
+In _Epic 40,000_ there are only ahead 90º, side 180º and 360º 'weapon fire arcs'.
```

Comment:
```suggestion
* diagonal left (180º); ahead (90º); front (180º); diagonal right (180º)
* left (180º); all round (360º); rear (180º); right (180º)

.Weapon arcs

[NOTE.e40k]
====
In _Epic 40,000_ there are only ahead 90º, side 180º and 360º 'weapon fire arcs'.
```

A couple of points, which I hadn't explained fully!


**Icons**
One of the main reasons behind the weapon arc icons was to remove the need to have a big list of unique names. In particular, if we want to use names, we're going to have to think through all the possible future weapon arcs, for example if 'rear' is already in use for a rear-facing 180º arc, what do we call a rear-facing 90º arc? 

This is obviously doable, but I'm not sure how much we'd gain from it. Also, I think it would have to be a direction+width construction. For example 'front-left 180º' and 'front-right 180º', as 'diagonal left' doesn't tell the reader if it's angled forwards or backwards.


**'Front' arc**
Second, if we're using names, 'front' is specifically the 90º forward-facing arc in E40k and is used on all the existing war engine data sheets and third-party data sheets, so changing it to 180º is... _risky_... ;-)

If we go down the route of unique names, we could use 'front 90º' and 'front 180º' or similar.

Reply:
Aaaaaand I'd missed '[TODO: We might want to agree on different terms and change those I've inserted below.]'...

Reply:
I don't think it is practical or desirable to not name the arcs at all. We can rely on the icons in the data sheets, but we'll often need to reference them by name in other places - and so we should define the names clearly and use them consistently where we need them. 

**Nomenclature**: Lol, Yes you spotted my note after your first comment. I just lobbed in some placeholders and added the note that we might want different terms. Still, I probably should've checked that I wasn't reallocating an existing term in my placeholders - the perils of dipping in and out.

Reply:
> we're going to have to think through all the possible future weapon arcs, for example if 'rear' is already in use for a rear-facing 180º arc, what do we call a rear-facing 90º arc?

Per my last comment, I don't think we can escape from the need for actual names for the arcs. Then yes, we should come up with a nomenclature that lends itself to extension, just in case, which I had in the back of my mind when I lobbed in placeholders and made a note.

Whether or not we should also have abbreviations is another debate, but they are arguably redundant when we will use icons on the data sheets.

>This is obviously doable, but I'm not sure how much we'd gain from it. Also, I think it would have to be a direction+width construction. For example 'front-left 180º' and 'front-right 180º', as 'diagonal left' doesn't tell the reader if it's angled forwards or backwards.

Direction+width construction: Yes. I was half-way there but didn't think it through to its conclusion in my quick placeholders. I have two possible approaches in mind:

**Option A: Unique by name, arc info in the definition only**
I favour this as you don't have to add the arc info each time and it is cleaner. It isn't *quite* fully self-descriptive, but I don't see this as a problem -- we've defined it, it is pretty easy to remember stuff for any wargamer, and the original content isn't fully self-descriptive.

In this topic only, for the full definition:
* Front-left (180º); Front (90º); Ahead (180º); Front-right (180º)
* Left (180º); All Round (360º); Behind (180º); Right (180º)
...and by extension Rear (90º).

So, outside of the full definition in this topic we'd actually use:
Front-left, Front; Ahead; Front-right; Left; All Round; Behind; Right
...and by extension Rear.

A further point for consideration in this option: We could emulate EA and call the 90 degree arcs 'Fixed X' to differentiate from all the others that, aside from All Round, are 180 degrees.

**Option B: Non-unique names, requires arc info to be unique**
This is fully self-descriptive, but more fiddly and clunky to say out loud.

* Front-left 180º; Front 90º; Front 180º; Front-right 180º
* Left 180º; All Round 360º; Rear 180º; Right 180º
...and by extension Rear 90º and Rear 180º.

*There is obviously some room to debate specific terms in both of these options too.*

Reply:
> **Option A: Unique by name, arc info in the definition only**
> I favour this as you don't have to add the arc info each time and it is cleaner. It isn't _quite_ fully self-descriptive, but I don't see this as a problem -- we've defined it, it is pretty easy to remember stuff for any wargamer, and the original content isn't fully self-descriptive either.

The original content didn't need to be fully self-descriptive, there were only four options! 

> In this topic only, for the full definition:
> 
> * Front-left (180º); Front (90º); Ahead (180º); Front-right (180º)
> * Left (180º); All Round (360º); Behind (180º); Right (180º)
>   ...and by extension Rear (90º).
> 
> So, outside of the full definition in this topic we'd actually use:
> Front-left, Front; Ahead; Front-right; Left; All Round; Behind; Right
> ...and by extension Rear.

The icons so far are just the current set of 'most-likely-to-be-used' weapon arcs, it's not a complete set of all possible options.

In total, there are **17** possible weapon arcs in the set. Eight directions * two widths, plus 360º.

I'm not sure a front-left 90º arc, left 90º arc or back-left 180º arc etc. would be _used_, but they need to be taken into account in any naming scheme.

For completeness, this is the 'full' set, using direction+width construction:
* Left 180º, Left 90º
* Front-left 180º, Front-left 90º
* Front 180º, Front 90º
* Front-right 180º, Front-right 90º
* Right 180º, Right 90º
* Back-right 180º, Back-right 90º (or Rear-right etc.)
* Back 180º, Back 90º (or Rear etc.)
* Back-left 180º, Back-left 90º (or Rear-left etc.)
* All Round 360º

---

> **Option B: Non-unique names, requires arc info to be unique**
> This is fully self-descriptive, but more fiddly, and somewhat clunky to say out loud.
> 
> * Front-left 180º; Front 90º; Front 180º; Front-right 180º
> * Left 180º; All Round 360º; Rear 180º; Right 180º
>   ...and by extension Rear 90º.
> 
> _There is obviously some room to debate specific terms in both of these options too._

If the angle of width is part of the name, which is what I was suggesting, that's a set of unique names.

I get your point that the full 'front-left one hundred and eighty degrees' name would be a tongue twister, but in practice it's going to be 'one-eighty to the corner' and point to the corner. 'Front-left 180º' in full form is only going to be used in text.

---

Going out of order a bit:
> A further point for consideration in this option: We could emulate EA and call the 90 degree arcs 'Fixed X' to differentiate from all the others that, aside from All Round, are 180 degrees.

I'd been wondering about that, but it already causes issues in EA because there are so many definitions of 'Fixed X arc' in different systems. I've come across the term being used for 22.5º, 45º and 90º arcs, and usually have to look it up if I've not played EA for a while.

The other issue is that we run into compatibility problems again if 'Front' by itself is 90º in E40k but is 180º in Remastered.

---

Summary - obviously it's your call at the end of the day, but a fully self-descriptive nomenclature is always going to get my vote if it's at all possible.

Reply:
Okay, I'm going to make some counter-points for you to consider, but these aren't meant to be combative and you'll see at the end that I'm willing to go your way if you still think that Option B (exemplified by your 'full st') is better.

>The original content didn't need to be fully self-descriptive, there were only four options!

Yes, but they aren't far off fully self-descriptive, and I don't think that we're creating a big cognitive burden here, which is the main point and therefore something less than 100% fully self-descriptive may be acceptable if it is less cumbersome in other ways.

>In total, there are 17 possible weapon arcs in the set. Eight directions * two widths, plus 360º.
>
>I'm not sure a front-left 90º arc, left 90º arc or back-left 180º arc etc. would be used, but they need to be taken into account in any naming scheme.

Yeah, exactly — it's nice to have something fully self-descriptive, but everything is a trade-off, so if we **know** that we probably won't use a lot of them anyway then it isn't as vital and other consideration **might** trump it. So in my view 'need' is far too strong a word there.

---

>If the angle of width is part of the name, which is what I was suggesting, that's a set of unique names.

Yes, that's my point, the 'name/label/descriptor' -- whatever you want to call the word part -- is not unique by itself, so you **must** add the arc/width/number part to make the complete 'name/term/whatever'. That's how I presented option B, and why I didn't use the brackets in that option.

---

>I'd been wondering about that, but it already causes issues in EA because there are so many definitions of 'Fixed X arc' in different systems. I've come across the term being used for 22.5º, 45º and 90º arcs, and usually have to look it up if I've not played EA for a while.

Okay, not a problem I've experienced but I can see how that can happen, I guess -- though how many people will really have to look it up to remind themselves that Epic doesn't even have 22.5 or 45 degree arcs, at all?

Of course there will always be some similar concepts and terms that are defined differently in other games -- so while we should consider such things, they can't be the be all and end all. 

Also, it's not like it has to be 'Fixed' -- it could be 'Narrow' or something?

>The other issue is that we run into compatibility problems again if 'Front' by itself is 90º in E40k but is 180º in Remastered.

Yes, that did cross my mind — but it doesn't seem like much of a compatibility problem if we just call it 'Fixed Front' does it? Assuming of course that we also use something like 'Ahead' rather than repurpose plain 'Front' for the front 180 arc.

---

All that said, if you don't think that my counterpoints above move the needle far enough in the direction of Option A then I'm happy to concede and go with Option B, exemplified by your 'full set' (I favour the 'rear' rather than 'back').

Reply:
Actually, scratch my last — I'll work in Option B.